### ISSUE TITLE
a problem about using dettrace

### ISSUE THREAD

**Issue by WorshipSmeb** [upenn-acg/detTrace#281]
when I continue learen using dettrace，I face a new problem。the description is as fellow：

first I built  the dettrace  on the /home/username/ Dettrace , and then use commands . /bin/dettrace -- make -C /home/username/dettrace-1 ， ./bin/dettrace - make -C /home /username/dettrace-2  , but when I used diffoscope to compare the executable file /home/username/dettrace-1/bin/dettrace and  /home/username/dettrace-2/bin/dettrace ,however they are not identical , and inconsistent information is the build directory was reported. I am using a virtual machine of ubuntu 18.04. on VM The kernel version is 5.4.0-52-generic x86_64 intel.

I don't use dettrace correctly or other things lead to this。I hope you can help me。thanks very much

**Comment by bmwiedemann** [comment]
You are varying the build path and it might be that dettrace cannot normalize that.
There is the `-fdebug-prefix-map` gcc compiler option that can help in such a case.

**Comment by gatoWololo** [comment]
Yes, by default Dettrace no longer creates a minimal lightweight container for builds (which is how we abstracted the build path, by bind mounting the CWD under `/build`). Since the way we did that was pretty fragile. The easiest way to use Dettrace inside a container with the same build path.
